### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SoJexy**
Add contributing guidelines for this open source project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contribution Guidelines for Procurify Open Source Projects
+Welcome and thank you for considering to contribute to Procurify's open source projects. We appreciate you stopping by and welcome you with open arms to be part of our community.
+
+Please take some time to read the following guidelines we've set to make the contribution process easy and accessible for everyone involved. 
+
+## Code of Conduct
+By participating and contributing to this project, you agree to uphold our [Code of Conduct](https://github.com/procurify/flag-counter/master/CODE-OF-CONDUCT.md).
```
Comment (rocpoon) [inline CONTRIBUTING.md:7]: Link is broken :D

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Contribution Guidelines for Procurify Open Source Projects
+Welcome and thank you for considering to contribute to Procurify's open source projects. We appreciate you stopping by and welcome you with open arms to be part of our community.
+
+Please take some time to read the following guidelines we've set to make the contribution process easy and accessible for everyone involved. 
+
+## Code of Conduct
+By participating and contributing to this project, you agree to uphold our [Code of Conduct](https://github.com/procurify/flag-counter/blob/main/CODE_OF_CONDUCT.md).
+
+## Getting started
+Please use the search to see if any exisitng Issues or PRs match your problem before creating your own
```
Comment (AndyHsiungDev) [inline CONTRIBUTING.md:10]: existing*

**Review by rocpoon — APPROVED** [review]
Looks good

**Review by AndyHsiungDev — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Getting started
+Please use the search to see if any exisiting Issues or PRs match your problem before creating your own
+
+We try to resolve issues in a timely manner, but if you feel like your issue is taking too long, feel free to leave a comment or ping a reminder to draw attention
+
+## Issue tracker
+If you wish to add changes to the core functionality of the project please email our development team at development@procurify.com
+
+If you find an issue that matches the one you're facing, feel free to comment if your environment differs from the one stated in the issue. Otherwise, adding a [reaction](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) is sufficient enough for use to know.
```
Comment (JustinTom) [inline CONTRIBUTING.md:17]: Small typo fix - "for us to know."

**Pull request merged**